config/secrets.yml

```diff
@@ -1,7 +1,5 @@
 default: &default
   secret_key_base: MGhvYnpIWnJaRHhLaDJrRkpOd1JqWFhHUHMuRFpVeGRhRE5GVHY1NDdBQkwxM0NkWGdnY1NnZmhNMWhCRzBBRG80U2YuTG5hMURrb3NkZ2VURkJOelZkZmhFMUM0TWJwdGFwcFJhM21hRHdQVTFoRUpWZXZCbmp3YnROZkZ0a2MK
-  http_auth_username: <%= ENV["SP_NAME"] %>
```

Comment:
Yeah, I don't mind where they go or get loaded from, just need to make sure we don't remove them from the app 😀